analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ConsoleLogging.cs

```diff
@@ -1,78 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
```

Comment:
Also these should remain, since S106 replaced S2228, they shared the test cases. Please revert the removal and maybe rename the files. to reflect which tests use them.

Reply:
Indeed - the name confused me. 
both Test files are using exactly the same test cases.
I just renamed them.